patches/components-net_log-resources-net_export.html.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/components/net_log/resources/net_export.html b/components/net_log/resources/net_export.html
+index 43ec9d24bd315744797ff13d70d01f8d2c9975fb..aeef55ccc2b35e1cfafdb4d666d8e92bccb8d7ac 100644
+--- a/components/net_log/resources/net_export.html
++++ b/components/net_log/resources/net_export.html
+@@ -47,7 +47,7 @@
+         Click the button to start logging future network activity to a file on
+         disk.
+         <if expr="not(is_ios)">
+-        The log includes details of network activity from all of Chrome,
```

Comment:
@petemill do we have some other way to handle Chrome -> Brave in files like this?

Reply:
I'm kind of surprised they have this static text in here anyway. Is this in English regardless of your locale settings? Maybe this needs an upstream patch to properly support localization